e2e/staticsite/staticsite_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package staticsite_test
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+)
+
+var _ = Describe("Static Site", func() {
+	Context("when creating a new app", Ordered, func() {
```

Comment:
I feel like they sort of don't need to be `Ordered` in this case! As long as `BeforeAll` is executed first, the `It` specs don't have to be ordered.
```suggestion
	Context("when creating a new app", func() {
```

Reply:
seems like it is required

```
  [BeforeAll] setup nodes must appear inside an Ordered container.  They cannot
  be nested within other containers, even containers in an ordered container.
```